common/remote/sockfile.cpp

```diff
@@ -3160,6 +3338,15 @@ class CClientStatsTable : public OwningStringSuperHashTableOf<CClientStats>
     }
 };
 
+struct OpenFileInfo
```

Comment:
minor/future: Code is fine as it is.  Chould possibly have this as a a linked structure, filename a StringAttr, and have a local instance of this which is filled in and then appended to a CArrayOf<> if the open succeeded.  As it is there will be 32 bytes being copied per file instead of 8 when 1 is removed.

Should possibly be a hash table or linked list( instead of an array since there will probably be quite a few entries, and elements get added and removed out of order.  (A std::map which uses a red-black tree may give the best of both worlds.)